lib/Doctrine/ODM/PHPCR/DocumentManagerInterface.php

```diff
@@ -489,6 +490,24 @@ public function findVersionByName($className, $id, $versionName);
     public function getUnitOfWork();
 
     /**
+     * Flushes all changes to objects that have been queued up to now to the database.
+     * This effectively synchronizes the in-memory state of managed objects with the
+     * database.
+     *
+     * This is different from the ObjectManager in that it accepts an optional
+     * argument to limit the flush to one or more specific documents.
+     *
+     * @param object|array|null $document optionally limit to a specific
+     *      document or an array of documents
+     *
+     * @return void
+     *
+     * @throws InvalidArgumentException if $document is neither null nor a
+     *      document or an array of documents
+     */
+    public function flush($document = null);
```

Comment:
i just adapted the .travis.yml to 5.3 instead of 5.3.3 - lets see. if its good, we should raise the php version in composer.json to 5.3.9 (i won't have a computer with me for the next 3 weeks...)